docs/references/strategies/languages/python/conda.md

```diff
@@ -9,7 +9,11 @@ Find the first file named `environment.yml`
 ## Analysis: conda list
 
 We run `conda list --json` in order to get a complete list of all installed conda packages in the conda environment.
+This tactic tries to determine which channel/platform a package was installed for to get the most accurate results.
```

Comment:
I made these additions to explain how this works, but I was on the fence on whether to include it. The behavior of fossa cli isn't very different from a user perspective, they'll mainly see changes on the back-end. On the other hand, this might be where someone ends up first looking for an explanation if they do see those changes.

Reply:
I'm a fan of including this, I think it makes this all clear and shows people that we're taking into account the location of each package. If users question how we know, we can point them here.